include/fmt/chrono.h

```diff
@@ -1560,6 +1580,38 @@ struct chrono_formatter {
     out = format_decimal<char_type>(out, n, num_digits).end;
   }
 
+  template <class Duration> void write_fractional_seconds(Duration d) {
+    static constexpr auto fractional_width =
```

Comment:
This doesn't need to be static because it's just an int.

Reply:
Good point, lightweight integral types probably do not need to be static, even if `constexpr`